File: trial/java/springboot-backend/src/main/java/dev/nazna/examplespringbackend/presentation/request/UserCreateRequest.java
package dev.nazna.examplespringbackend.presentation.request;

import lombok.Data;
import lombok.experimental.Accessors;

import javax.validation.constraints.Max;
import javax.validation.constraints.NotBlank;

@Data
@Accessors(chain = true)
public class UserCreateRequest {

  @NotBlank
  @Max(16)
  public String name;

}

File: example-multilang-monorepo/backend/src/main/kotlin/dev/nazna/examplemonorepoapp/presentation/request/UserCreateRequest.kt
package dev.nazna.examplemonorepoapp.presentation.request

import kotlinx.serialization.Serializable
import javax.validation.constraints.NotBlank
import javax.validation.constraints.Size

@Serializable
data class UserCreateRequest(
  @field:NotBlank
  @field:Size(max = 32)
  val name: String
)
